api_v3/lib/types/KalturaUserEntry.php

```diff
@@ -55,6 +56,12 @@
 	 */
 	public $updatedAt;
 
+	/**
+	 * @var KalturaUserEntryType
+	 * @readonly
+	 * @filter eq
+	 */
+	public $type;
```

Comment:
I don't know why you expose the type, if it's just for the filter, then make the KalturaUserEntryFilter abstract, anyway it doesn't make sense to search more than one type in the same request.